code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -10,6 +10,7 @@
 	clothing_flags = HAS_UNDERWEAR | HAS_UNDERSHIRT | HAS_SOCKS
 	bodyflags = HAS_SKIN_TONE | HAS_BODY_MARKINGS
 	dietflags = DIET_OMNI
+	taste_sensitivity = TASTE_SENSITIVITY_DULL
```

Comment:
See [this line](https://github.com/ParadiseSS13/Paradise/pull/7333/files#diff-aecbf391c938576bbf2e44885a709d7aR48). Why are Humans redefined with dull taste here when it explicitly says that the variable definition (its default value) is the factor of how much reagent needed for a human to taste it?